testing/cli_tests/constraint.py

```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/env python3
```

Comment:
It would be great to test this in the Rust `tests/integration/fuzz` suite, and run it thousands of iterations per primary key, for both limbo and sqlite. can be done later though, but maybe a comment here about it would be good